test/JDBC/input/BABEL-1661.mix

```diff
@@ -0,0 +1,108 @@
+-- psql
```

Comment:
Let's not use `BABEL-1661` as a test file name. We are trying to give more meaningful test file name.